cmd/kwild/server/build.go

```diff
@@ -768,20 +768,30 @@ func buildStatesyncer(d *coreDependencies, db sql.ReadTxMaker) *statesync.StateS
 
 	providers := strings.Split(d.cfg.ChainConfig.StateSync.RPCServers, ",")
 
-	if len(providers) == 0 {
+	if len(providers) == 0 || d.cfg.ChainConfig.StateSync.RPCServers == "" {
```

Comment:
Oh good catch.  I'm not sure how it would ever be `len(providers) == 0`, but might as well keep that.